package.json

```diff
@@ -57,10 +57,8 @@
   ],
   "license": "BSD-3-Clause",
   "lint-staged": {
-    "./*.js": "npm run lint-arg --",
     ".eslintignore": "npm run lint",
-    "src/**/*.js": "npm run lint-arg --",
-    "test/**/*.js": "npm run lint-arg --"
+    "*.js": "eslint --fix"
```

Comment:
I'd prefer not to have these changes. I like to refer to `lint-arg` so we can keep the actual linting directives in one place (including calling `--report-unused-disable-directives`).

I am ok, however, with reducing it to a single line pointing to `npm run lint-arg --` if @gajus doesn't mind us linting all files by default, leaving the ignore file to settle which ones should be ignored.

Reply:
Yeah, `--report-unused-disable-directives` is useful, so I added it to the call, and just dropped the `lint-arg` script

Reply:
> `lint-staged` only passes the touched files, and skips them if they're ignored by the `.eslintignore`.

Ah yes, sorry, forgot you were actually not getting rid of lint-staged either.

> I think the extra `.eslintignore` script you have is a neat idea, but I think it might be missing a call to `git add .`. `lint-staged` will auto-add the files it touches, but won't for the extra files that your larger call flags

The idea is merely that we should relint everything, and although `lint-staged` doesn't pass in all files as part of its file args (such as we leverage with our `lint-arg` calls), in running our `npm run lint` with `.eslintignore`, that script should, I believe, cause *all* files to be re-linted due to the nature of `npm run lint` running everything.

Reply:
Right, I was mixing up the current call to `lint` instead of `lint-fix`, which might have changes files to re-add 😄

Reply:
Ah. Well, I guess that's not a bad idea, so welcome to apply `lint-fix` for the ignore file too.

But as mentioned in the review comments, I do want to reference npm scripts.